deploy/docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+cassandra:
+  image: itszero/zipkin-cassandra
+collector:
+  image: itszero/zipkin-collector
+  expose:
+    - 9410
+  ports:
+    - 9410:9410
+  links:
+    - cassandra:db
+query:
+  image: itszero/zipkin-query
+  expose:
+    - 9411
+  links:
+    - cassandra:db
+web:
+  image: itszero/zipkin-web
+  links:
+    - query
+  ports:
+    - 8080:8080
+  environment: {
+    ROOTURL: "http://172.17.42.1:8080"
```

Comment:
Yep, it's fine if I remove it. I never looked into what ROOTURL is used for, I just assumed [this](https://github.com/openzipkin/docker-zipkin/blob/master/deploy/deploy.sh#L5) is there for a good reason :)